share/scudcloud.desktop

```diff
@@ -6,7 +6,6 @@ Categories=Network;Chat;
 Name=ScudCloud
 Comment=ScudCloud is an open-source client for Slack
 Keywords=slack;chat;instant-messaging;collaboration;
-Exec=/opt/scudcloud/scudcloud
+Exec=scudcloud
 StartupWMClass=scudcloud
-Icon=/opt/scudcloud/resources/scudcloud.png
-
+Icon=/usr/share/pixmaps/scudcloud.png
```

Comment:
I know it's not pretty, but the problem is each IDE will search for the `scudcloud` file, it not exactly specified, in a different order.

Some distros will search first on `/usr/share/icons/hicolor`, Ubuntu will search first in `/usr/share/icons/ubuntu-mono-dark` (or `light`).

So, to avoid the problem, I've changed setup to copy the file to `/usr/share/pixmaps`, as other apps use this folder too.

And, for the `.desktop` file will accept just the name without extension (like `scudcloud`), the full path (like this change), or the mime (if the app registered one).